components/textbox/textbox_links/textbox_links.tsx

```diff
@@ -2,141 +2,145 @@
 // See LICENSE.txt for license information.
 
 import React, {MouseEvent} from 'react';
+import {useSelector} from 'react-redux';
 import {FormattedMessage} from 'react-intl';
 import {Link} from 'react-router-dom';
 
+import {getCurrentLocale} from 'selectors/i18n';
+
 type Props = {
     showPreview?: boolean;
     characterLimit: number;
     previewMessageLink?: string;
     updatePreview?: (showPreview: boolean) => void;
-    message: string;
+    message?: string;
     isMarkdownPreviewEnabled: boolean;
 };
 
-export default class TextboxLinks extends React.PureComponent<Props> {
-    static propTypes = {
-    };
+const TextboxLinks: React.FC<Props> = ({
+    showPreview,
+    characterLimit,
+    previewMessageLink,
+    updatePreview,
+    message = '',
+    isMarkdownPreviewEnabled,
+}: Props) => {
+    const currentLocale: string = useSelector(getCurrentLocale);
```

Comment:
Instead of using `useSelector`, could you pass in the current locale from the `mapStateToProps` in the index file instead so that we're not mixing ways of accessing the store for this component?

Reply:
Sure thing, addressed! 😊